plugins/services/src/js/containers/pod-detail/PodDetail.js

```diff
@@ -18,14 +18,14 @@ import ServiceActionLabels from "../../constants/ServiceActionLabels";
 import ServiceBreadcrumbs from "../../components/ServiceBreadcrumbs";
 import ServiceModals from "../../components/modals/ServiceModals";
 import ServiceTree from "../../structs/ServiceTree";
-import {
-  DELETE,
-  EDIT,
-  OPEN,
-  RESUME,
-  SCALE,
-  STOP
-} from "../../constants/ServiceActionItem";
+import { ServiceActionItem } from "../../constants/ServiceActionItem";
+
+const DELETE = ServiceActionItem.DELETE;
+const EDIT = ServiceActionItem.EDIT;
+const OPEN = ServiceActionItem.OPEN;
+const RESUME = ServiceActionItem.RESUME;
+const SCALE = ServiceActionItem.SCALE;
+const STOP = ServiceActionItem.STOP;
```

Comment:
Same question with reassigning to intermediate const

Reply:
I find that easier and clearer instead of having `ServiceActionItem.SOMETHING` in the code. It's also more similar to what we had before.